Article/Article.js

```diff
@@ -2,22 +2,39 @@
 
 class Article {
   constructor(domElement) {
+//parameter
     // assign this.domElement to the passed in domElement
-    this.domElement;
+
+    this.domElement = domElement;
+
+
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+
+    this.expandButton = domElement.querySelector('.expandButton');
+
+
     // Using your expandButton reference, update the text on your expandButton to say "expand"
     
+    this.expandButton.textContent = 'expand';
+
+
     // Set a click handler on the expandButton reference, calling the expandArticle method.
     
+    this.expandButton.addEventListener('click', () => this.expandArticle());
+
   }
 
   expandArticle() {
+    this.domElement.classList.toggle('article-open');
```

Comment:
In here, try to make it so that, whenever you expand your article, instead of `expand` it says `close`